python_modules/dagster-graphql/dagster_graphql/implementation/execution/backfill.py

```diff
@@ -114,17 +114,22 @@ def create_and_launch_partition_backfill(
     partitions_by_assets = backfill_params.get("partitionsByAssets")
 
     if (
-        asset_selection or backfill_params.get("selector") or backfill_params.get("partitionNames")
+        asset_selection
+        or backfill_params.get("selector")
+        or backfill_params.get("partitionNames")
+        or backfill_params.get("allPartitions")
     ) and partitions_by_assets:
         raise DagsterInvariantViolationError(
-            "partitions_by_assets cannot be used together with asset_selection, selector, or"
-            " partitionNames"
+            "partitionsByAssets cannot be used together with assetSelection, selector, or"
+            " partitionNames or if allPartitions is True"
         )
 
     tags = {t["key"]: t["value"] for t in backfill_params.get("tags", [])}
 
     tags = {**tags, **graphene_info.context.get_viewer_tags()}
 
+    title = check_valid_title(backfill_params.get("title"))
```

Comment:
It seems reasonable to use the existing utility. Also, I think there's no reason it should only be happening for job backfills.

Reply:
doing it for only job backfills is 100% just a goof on my part. we should be doing it for all types of backfills - thanks for catching! 

switching to the check version of the fn is also fine w me. I think originally we were considering re-raising as a GrapheneError so that it had a more immediate failure for the user. But we've kind of abandoned adding titles/descriptions to backfills for now anyway, so keeping it as the generic python error is fine

Reply:
@jamiedemaria Great! I raised a new PR just addressing this (tagged you, if you can take a quick look), since it's agreed upon, while I go about changing this broader PR.